images/win/scripts/ImageHelpers/ChocoHelpers.ps1

```diff
@@ -29,4 +29,21 @@ function Choco-Install {
             }
         }
     }
+}
+
+function Get-ChocoPackageLatestVersion {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $PackageName,
+        [Parameter(Mandatory)]
+        [string] $TargetVersion
+    )
+
+    $filterQuery = "`$filter=(Id eq `'$PackageName`') and (IsPrerelease eq false) and (Version ge `'$TargetVersion`') and (Version lt `'$($TargetVersion + 1)`')"
+    $latestVersion = (Invoke-RestMethod "https://community.chocolatey.org/api/v2/Packages()?$filterQuery").properties.Version |
+        Sort-Object{[version]$_} -Descending |

```

Comment:
Add a space after Sort-Object

Reply:
Remove -Descending and use Last instead of First

Reply:
> Remove -Descending and use Last instead of First

Packages are not sorted properly so sorting here is necessary.

Reply:
I mean remove only the `-Descending` param and not entire Sort-Object.

Reply:
> I mean remove only the `-Descending` param and not entire Sort-Object.

I see. Will update.

Reply:
Done